agent/src/beerocks/slave/platform_manager/platform_manager_thread.cpp

```diff
@@ -126,6 +130,8 @@ static bool fill_platform_settings(
         BPL_BACK_VAPS_GROUPS * BPL_BACK_VAPS_IN_GROUP * BPL_MAC_ADDR_OCTETS_LEN;
     char back_vaps[back_vaps_buff_len];
 
+    int temp_int;
```

Comment:
I defined it here since, during the development of the unified Agent, all parameters will be moved eventually to the DB.
So I preferred to define it here so I won't have to move it each time we'll move another parameter to the database.